.gitignore

```diff
@@ -41,6 +41,7 @@ node_modules
 
 # IntelliJ
 .idea
+*.class
```

Comment:
Arguably, this change has nothing to do with this PR, but I don't suppose it matters too much.